diff --git a/chrome/browser/extensions/api/developer_private/extension_info_generator.cc b/chrome/browser/extensions/api/developer_private/extension_info_generator.cc
index 4aec8139d3fc8..eec6028ac5bde 100644
--- a/chrome/browser/extensions/api/developer_private/extension_info_generator.cc
+++ b/chrome/browser/extensions/api/developer_private/extension_info_generator.cc
@@ -871,17 +871,6 @@ void ExtensionInfoGenerator::FillExtensionInfo(const Extension& extension,
         toolbar_actions_model->IsActionPinned(extension.id());
   }
 
-  // MV2 deprecation.
-  ManifestV2Handler* mv2_handler = ManifestV2Handler::Get(profile);
-  CHECK(mv2_handler);
-  info.is_affected_by_mv2_deprecation =
-      mv2_handler->IsExtensionAffected(extension);
-  if (info.web_store_url.length() > 0) {
-    info.recommendations_url =
-        extension_urls::GetNewWebstoreItemRecommendationsUrl(extension.id())
-            .spec();
-  }
-
   info.can_upload_as_account_extension =
       AccountExtensionTracker::Get(profile)->CanUploadAsAccountExtension(
           extension)
diff --git a/chrome/browser/ui/webui/extensions/extensions_ui.cc b/chrome/browser/ui/webui/extensions/extensions_ui.cc
index b8bec16f630e5..441f63d88e52b 100644
--- a/chrome/browser/ui/webui/extensions/extensions_ui.cc
+++ b/chrome/browser/ui/webui/extensions/extensions_ui.cc
@@ -486,7 +486,7 @@ content::WebUIDataSource* CreateAndAddExtensionsSource(Profile* profile,
           extensions_features::kExtensionsMenuAccessControlWithPermittedSites));
   source->AddBoolean(
       "safetyHubThreeDotDetails",
-      base::FeatureList::IsEnabled(features::kSafetyHubThreeDotDetails));
+      false);
   source->AddBoolean("enableGlobalScopedShortcuts", IsGlobalShortcutEnabled());
 
   // MV2 deprecation.
diff --git a/extensions/browser/manifest_v2_handler.cc b/extensions/browser/manifest_v2_handler.cc
index 9f7c55596044b..787d6beee95dd 100644
--- a/extensions/browser/manifest_v2_handler.cc
+++ b/extensions/browser/manifest_v2_handler.cc
@@ -113,7 +113,7 @@ bool ShouldDisableLegacyExtensions() {
     return false;
   }
-
+
-  return true;
+  return false;
 }
-
+
 }  // namespace
diff --git a/extensions/common/extension.cc b/extensions/common/extension.cc
index 7e3abea6cb2d5..99dbb8ee585d1 100644
--- a/extensions/common/extension.cc
+++ b/extensions/common/extension.cc
@@ -100,12 +100,6 @@ bool IsManifestSupported(int manifest_version,
   // Supported versions are always safe.
   if (manifest_version >= kMinimumSupportedManifestVersion &&
       manifest_version <= kMaximumSupportedManifestVersion) {
-    // Emit a warning for unpacked extensions on Manifest V2 warning that
-    // MV2 is deprecated.
-    if (type == Manifest::Type::kExtension && manifest_version == 2 &&
-        Manifest::IsUnpackedLocation(location)) {
-      *warning = errors::kManifestV2IsDeprecatedWarning;
-    }
     return true;
   }
 
